scripts/plugin-jsdocs.js

```diff
@@ -1,10 +1,10 @@
 const path = require('path');
 const fsp = require('fs-promise');
-const esprima = require('esprima');
+const espree = require('espree');
 
 // Parse the registry and extract the class methods, parameters and leading comments.
 const registryContent = fsp.readFileSync(path.join(__dirname, 'mattermost-webapp/plugins/registry.js'), 'utf-8')
-const registryParsed = esprima.parseModule(registryContent, { comment: true, attachComment: true });
+const registryParsed = espree.parse(registryContent, { comment: true, attachComment: true, sourceType: 'module', ecmaVersion: 10 });
```

Comment:
I set the `ecmaVersion` as `10` because it works (it also works with 9, but not with 8 or older). But I don't really know what we should set it to.